packages/rfw/example/wasm/lib/main.dart

```diff
@@ -14,13 +14,16 @@ import 'package:path_provider/path_provider.dart';
 import 'package:rfw/rfw.dart';
 import 'package:wasm/wasm.dart';
 
-const String urlPrefix = 'https://raw.githubusercontent.com/flutter/packages/master/packages/rfw/example/wasm/logic';
+const String urlPrefix =
```

Comment:
You should revert the formatting changes to this file; Hixie opted it out of `dartfmt`.